gossip/src/cluster_info.rs

```diff
@@ -671,6 +671,13 @@ impl ClusterInfo {
         self.push_self();
     }
 
+    pub fn set_tpu(&self, tpu_addr: SocketAddr) {
+        let _ = self.my_contact_info.write().unwrap().set_tpu(tpu_addr);
```

Comment:
can't ignore the error here

Reply:
@behzadnouri Could you please check commit [1f17f4f](https://github.com/solana-labs/solana/pull/30452/commits/1f17f4f523429866ef253ec18c497d533588e52a)? I hope I understood you correctly.